pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/tenant/TenantRebalanceConfig.java

```diff
@@ -71,6 +88,25 @@ public void setParallelBlacklist(Set<String> parallelBlacklist) {
     _parallelBlacklist = parallelBlacklist;
   }
 
+  @ApiModelProperty(hidden = true)
+  public Set<String> getAllowTables() {
+    return _allowTables;
+  }
+
+  public void setAllowTables(Set<String> allowTables) {
+    _allowTables = allowTables;
+  }
+
+  @ApiModelProperty(hidden = true)
+  public Set<String> getBlockTables() {
+    return _blockTables;
+  }
+
+  public void setBlockTables(Set<String> blockTables) {
+    _blockTables = blockTables;
+  }
+
+  @ApiModelProperty(hidden = true)
```

Comment:
Yet to look at the full code in detail, but will marking these as `hidden = true` break backward compatibility for users that use these properties?